policyengine_us/variables/gov/states/mt/tax/income/exemptions/personal/mt_personal_exemptions_indiv.py

```diff
@@ -0,0 +1,31 @@
+from policyengine_us.model_api import *
+
+
+class mt_personal_exemptions_indiv(Variable):
```

Comment:
Could you add one unit test for each of the file variables which include the `p.applies` structure, for the year where the logic changes